packages/legend-application-studio/src/components/workspace-setup/WorkspaceSetup.tsx

```diff
@@ -182,6 +183,14 @@ export const WorkspaceSetup = withWorkspaceSetupStore(
       }
     };
 
+    const handleEditProject = (): void => {
```

Comment:
remember this is not `edit project`, this is more like `manage project` or `view project dashboard`